docs/content/code-generator.md

```diff
@@ -13,7 +13,7 @@ The Unity GDK comes packaged with a custom code generator. This doc will detail
 
 The code generator operates on the [JSON AST](https://docs.improbable.io/reference/latest/shared/schema/custom-codegen#abstract-syntax-tree-format) that is generated by the schema compiler. A pre-built `dll` parses this JSON AST and converts it to C# objects.
 
-This parsing stage can be found in [CodeGenerator.cs](../../code_generator/src/CodeGenerator.cs):
+This parsing stage can be found in [CodeGenerator.cs](../../code_generator/GdkCodeGenerator/src/CodeGenerator.cs):
 
```

Comment:
"..can be found..." is passive voice (see [Docs space in Confluence](https://improbableio.atlassian.net/wiki/spaces/GBU/pages/220725249?atlOrigin=eyJpIjoiNWZjYjgzNjIyZjdjNDQzM2IxMGRkMGNlNDJhMmY5OWUiLCJwIjoiYyJ9). Better to make it active voice:
"The parsing stage is in [CodeGenerator.cs](../../code_generator/src/CodeGenerator.cs):"

Reply:
Also, I think the code snippet is the parsing stage, in which case, we should say so.
""The parsing stage, shown in the code snippet below, is in [CodeGenerator.cs](../../code_generator/src/CodeGenerator.cs),:"